lib/components/create_join_dialog.dart

```diff
@@ -221,9 +221,10 @@ class CreateJoinBeaconDialog {
                               model.startingTime.minute,
                             );
                             // localNotif.scheduleNotification();
-                            if (model.startsAt.isBefore(DateTime.now())) {
-                              navigationService.showSnackBar(
-                                  "Enter a valid date and time!!");
+                            if (model.startsAt.isBefore(DateTime.now()
+                                .subtract(Duration(minutes: 1)))) {
+                              navigationService
+                                  .showSnackBar("Enter a valid date and time!");
```

Comment:
Instead of snack bar, I guess it would be better to add validations in the fields itself. When keyboard is on snackbar popups aren't displayed well.

Reply:
Ok! So should I close the keyboard upon this error and then show snackbar or validate the field itself?

Reply:
Validating fields would be better.

Reply:
Any progress? @ItsAdityaKSingh